packages/paste-core/components/breadcrumb/__tests__/breadcrumb.spec.tsx

```diff
@@ -66,4 +67,206 @@ describe('Breadcrumb', () => {
       expect(results).toHaveNoViolations();
     });
   });
+
+  describe.only('HTML attributes', () => {
```

Comment:
Hmm this describe block name is a little misleading because we're checking for data-attributes specifically for customization. Maybe consider moving this into the customization block?

Reply:
This is how most of the customization tests have been done, was following the patterns we established. I agree though that the describe block is confusing, I think though it warrants a separate ticket/effort to address in all the customization tests.

Reply:
np